<mat-card appMaterialElevation defaultElevation="4" raisedElevation="4">
  <p>
    Extracellular vesicles (EVs) released by live cells are important vectors for signal transduction and cell communication, especially for
    the interactivity between distant cells. Exosomes secreted from endosomal system and microvesicles (MVs) shed from plasma membrane are
    the two main sources of EVs. The contents of EVs comprise multiple kinds of biomolecules from source cells, such as lipid, protein,
    metabolite, DNA and RNA. Non-coding RNAs (ncRNAs) are well-studied and important molecules in EVs, which can be delivered into and
    regulate target cells, and show a physiological- and pathological-specificity to serve as candidate biomarkers.
  </p>
  <p>
    Extracellular Vesicle Atlas (EVAtlas) is an update of
    <a href="http://bioinfo.life.hust.edu.cn/EVmiRNA#!/" target="_blank">EVmiRNA</a> database. We performed a unified read dynamic
    assignment algorithm (RDAA) considering mismatch and multi-mapping reads to quantify the expression profiles of seven ncRNA types
    (miRNA, snoRNA, piRNA, snRNA, rRNA, tRNA and Y RNA) in more than 2,030 high-quality small RNA-seq datasets of EV samples from 24 human
    tissues of different conditions (disease and normal). EVAtlas provides four functional modules: (i) browse and compare the distribution
    of ncRNAs in EVs from 24 conditions and eight sources (plasma, serum, saliva, urine, sperm, breast milk, primary cell and cell line );
    (ii) prioritize candidate ncRNAs in condition related tissues based on their expression; (iii) explore the specifically expressed ncRNAs
    in EVs from 24 conditions; (iv) investigate ncRNA functions, related drugs, target genes and EVs isolation methods.
  </p>
</mat-card>
