<div fxLayout="column" fxFill>
  <div fxFlex="none">
    <h1 class="page-header">
      Document
    </h1>
  </div>

  <div fxFlex class="not-float">
    <mat-card appMaterialElevation defaultElevation="4" raisedElevation="4">
      <mat-card-header>
        <mat-card-title>Introduction</mat-card-title>
      </mat-card-header>
      <mat-card-content>
        <p>
          EVAtlas (Extracellular vesicle Atlas) is a comprehensive database depositing
          <span style="color: DarkViolet;">
            expression profiles of 8 types of ncRNAs (miRNA, piRNA, rRNA, tRNA, scRNA, snRNA, snoRNA, pRNA)</span>
          in more than 2,000 EV samples from 22 human tissues(organs) and 40 diseases. In addition, EVAtlas provides
          modules for user to
          explore the conditional expression profiles of ncRNAs, miRNA targets, miRNA related drugs, miRNA regulated
          pathways, miRNA
          functions, and related publications.
        </p>
        <h3>Abstract Summary</h3>
        <img mat-card-image class="zm" [src]="assets + '/imgs/' + 'zm.png'" />
      </mat-card-content>
    </mat-card>
    <br />

    <mat-card appMaterialElevation defaultElevation="4" raisedElevation="4">
      <mat-card-header>
        <mat-card-title>Function modules</mat-card-title>
      </mat-card-header>
      <mat-card-content>
        <h3></h3>
        <ol>
          <li>
            <p>
              <a routerLink="/samples">Sample module:</a> helps users to search the project and sample information of
              Exosomes or
              Microvesicles in tissues , and browse their component ratio, heatmap and expression profiles of 8 types of
              ncRNAs.
            </p>
          </li>
          <li>
            <p>
              <a routerLink="/rna">Expression module:</a> display the expression distribution of ncRNAs in collected
              samples and tissues,
              and provides the basic information and expressin profiles for ncRNAs across samples and conditions. For
              miRNAs, users can
              browsed comprehensive miRNA infomation, contain miRNA basic infomation, miRNA expression profiles, miRNA
              related drug, miRNA
              regulated pathways, miRNA function, and related publications.
            </p>
          </li>

          <li>
            <p>
              <a routerLink="/specific">Specific Expression module:</a> users can browse the specific expression
              patterns of ncRNAs(miRNA,
              piRNA and others) in tissues and diseases.
            </p>
          </li>

          <li>
            <p>
              <a routerLink="/">Search module:</a> provides a quick search function for users to rapidly inquery the
              expression profiles of
              their interested ncRNAs across samples and conditions.
            </p>
          </li>
        </ol>
      </mat-card-content>
    </mat-card>
    <br />

    <div fxLayout.lt-lg="column" fxLayoutGap="10px">
      <mat-card fxFlex appMaterialElevation defaultElevation="4" raisedElevation="4">
        <mat-card-header>
          <mat-card-title>{{ exoMappingRateTitle }}</mat-card-title>
        </mat-card-header>
        <mat-card-content>
          <div *ngIf="!exoMappingRate">
            <mat-progress-bar mode="query" color="warn"></mat-progress-bar>
          </div>
          <div echarts [options]="exoMappingRate" class="chart-hight"></div>
        </mat-card-content>
      </mat-card>
      <br />

      <mat-card fxFlex appMaterialElevation defaultElevation="4" raisedElevation="4">
        <mat-card-header>
          <mat-card-title>{{ mvMappingRateTitle }}</mat-card-title>
        </mat-card-header>
        <mat-card-content>
          <div class="spinner-container" *ngIf="!mvMappingRate">
            <mat-progress-bar mode="query" color="warn"></mat-progress-bar>
          </div>
          <div echarts [options]="mvMappingRate" class="chart-hight"></div>
        </mat-card-content>
      </mat-card>
    </div>
    <br />

    <mat-card appMaterialElevation defaultElevation="4" raisedElevation="4">
      <mat-card-header>
        <mat-card-title>Data Summary</mat-card-title>
      </mat-card-header>
      <mat-card-content>
        <ol>
          <li>
            <p>
              We downloaded a total of 2,831 small RNA sequencing datasets of exosomes and microvesicles samples
              <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_blank">NCBI SRA database</a> (up to 30/05/2020).
              Then, we manually
              examined the sources of samples and their isolation method to ensure their categories (exosome or
              microvesicle or others)
              according to Mateescu's paper (details in the following picture). Finally, after quality control, we
              obtained
              <strong style="color: DarkViolet;">2,271</strong> samples from 23 human tissues(organs) and 40 diseases.
              Furthermore,.
            </p>
            <h3>EV isolation techniques</h3>
            <img mat-card-image class="evi_func" [src]="assets + '/imgs/' + 'evisolated.jpg'" />
            <p>
              <cite>
                Mateescu, Bogdan, et al. "Obstacles and opportunities in the functional analysis of extracellular
                vesicle RNA-an ISEV
                position paper." Journal of extracellular vesicles 6.1 (2017): 1286095.</cite>
            </p>
            <ul>
              <li>
                (a) Differential centrifugation is the sequential pelleting of particles with decreasing sedimentation
                coefficients.
                Typically 2000 g is used to pellet large EVs, 10,000-20,000 g to pellet middle-sized EVs (green) (MVs),
                and finally ~100,000
                g to pellet the smallest EVs (different EV subpopulations are indicated in grey and orange) (Exosomes).
                At these high
                g-forces, complexes of soluble proteins (black dots) may also sediment.
              </li>
              <li>
                (b) Lipids have a density that is approximately 1 g cm-3, while proteins and RNA have a higher density
                (>1.3 g cm-3).
                Therefore density gradients can be used to separate subpopulations of EVs with different ratio of
                lipids, RNA, and proteins.
                Moreover, these gradients can be used to purify vesicles away from soluble proteins, RNA, and
                protein-RNA complexes as the
                latter structures will not float at the same density as the lipid containing EVs.
              </li>
              <li>
                (c) Size exclusion chromatography separates particles based on their size, by trapping the smaller
                molecules (such as
                proteins and protein complexes) in the pores. The larger molecules (such as EVs) are too large to enter
                the pores and will
                elute first.
              </li>
              <li>
                (d) Precipitation of EV from cell culture medium or body fluids is based on volume-excluding polymers
                such as polyethylene
                glycol (PEG) with which biological materials such as proteins and EVs are precipitated from the
                solution.
              </li>
              <li>
                (e) (Immuno-)affinity capture isolates vesicles using beads coated with antibodies or proteins (such as
                heparin) with
                affinity for an EV transmembrane protein. Vesicles displaying the protein of interest will bind to the
                beads and can thereby
                be isolated from the vesicle-containing solution.
              </li>
            </ul>

            <p><b>Small RNA sequencing data mapping Strategy</b></p>
            <p>
              Datasets with ambiguous description of experiment design and less raw reads were discarded. Subsequently,
              for each dataset,
              reads with the multiple N bases (>1) or unexpected length after adapter remove (> 45 nt or < 15 nt) or
                with low quality (mean quality <=20) were filtered, and dataset with less reads (< 1M) after the
                procedure were discarded. Finally, 2271 EV datasets were kept for further analysis, which including
                1,660 exosome and 563 MV samples from 23 human tissues(organs) and 40 diseases (Table 1). Clean reads
                were mapped to human pre-miRNA sequences from miRBase V22 and other ncRNAs from Genebank using bowtie (1
                mismatch allowed) to identify known ncRNAs and estimate their abundance. The normalization method of RPM
                (expressed Reads Per Million reads) were employed to assess the expression level of ncRNAs across
                samples and conditions. </p>
          </li>
          <br />
          <li>
            <p>
              Annotation data source: The annotation file of piRNAs was downloaded from piRNAQuest database, which the
              snoRNA annotation was
              obtained from snoRNABase. The annotation information of snRNA, scRNA and pRNA were parsed from ENSEMBL v86
              gtf file. The rRNA
              and tRNA information were incubated from NCBI GeneBank database. The miRNA basic information was collected
              from miRbase.
              Pathway information was collected from KEGG database, and use Hypergeometric p-value to filter miRNA
              pathway pairs.
              Publication information was collected from NCBI pubmed database. miRNA function was collected from NCBI
              RIF database. miRNA
              Drug information was collected from SM2miR (Small molecular miRNA database). miRNA targets was from
              TargetScan, miRanda,
              TarBase, miRTarBase, miR2Disease, miRecords and miRWalk.
            </p>
            <b>RNA types & annotation</b>
            <img mat-card-image class="df" [src]="assets + '/imgs/' + 'data_function_static.png'" />
          </li>
          <li>
            <p>
              We applied SEGtool to identify Specific miRNAs of cancer and tissue. We used miRNA expression cutoff RPM
              >= 100 to identify
              specific miRNAs in cancer group and tissue group. We found that in cancer group, Exosome can identify more
              specific miRNAs.
            </p>
            <p>
              <b>Specific miRNA</b>
            </p>
            <table mat-table [dataSource]="dataSource" class="mat-elevation-z8">
              <ng-container matColumnDef="Source">
                <th mat-header-cell *matHeaderCellDef>No.</th>
                <td mat-cell *matCellDef="let element">{{ element.Source }}</td>
              </ng-container>

              <ng-container matColumnDef="EV_type">
                <th mat-header-cell *matHeaderCellDef>EV type</th>
                <td mat-cell *matCellDef="let element">{{ element.EV_type }}</td>
              </ng-container>

              <ng-container matColumnDef="Cancer_Source_type">
                <th mat-header-cell *matHeaderCellDef>Source type</th>
                <td mat-cell *matCellDef="let element">{{ element.Cancer_Source_type }}</td>
              </ng-container>

              <ng-container matColumnDef="Specific_miRNA_counts">
                <th mat-header-cell *matHeaderCellDef>Specific miRNA counts</th>
                <td mat-cell *matCellDef="let element">{{ element.Specific_miRNA_counts }}</td>
              </ng-container>

              <ng-container matColumnDef="Specific_piRNA_counts">
                <th mat-header-cell *matHeaderCellDef>Specific piRNA counts</th>
                <td mat-cell *matCellDef="let element">{{ element.Specific_piRNA_counts }}</td>
              </ng-container>

              <ng-container matColumnDef="Specific_others_counts">
                <th mat-header-cell *matHeaderCellDef>Specific others counts</th>
                <td mat-cell *matCellDef="let element">{{ element.Specific_others_counts }}</td>
              </ng-container>

              <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
              <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
            </table>
          </li>
        </ol>
      </mat-card-content>
    </mat-card>
  </div>

  <div fxFlex="none">
    <app-footer></app-footer>
  </div>
</div>