<div fxLayout="column" fxLayoutGap="10px" fxFill>
  <div fxFlex="none">
    <h1 class="page-header">Document</h1>
  </div>

  <!-- Introduction -->
  <div fxFlex fxLayout="column" fxLayoutGap="10px" class="not-float">
    <mat-card fxFlex appMaterialElevation defaultElevation="4" raisedElevation="4">
      <mat-card-header>
        <mat-card-title>Introduction</mat-card-title>
      </mat-card-header>
      <mat-card-content>
        <p>
          Extracellular Vesicle Atlas (EVAtlas) is a comprehensive database depositing
          <span style="color: DarkViolet;">
            expression profiles of seven types of ncRNAs (miRNA, snoRNA, piRNA, snRNA, rRNA, tRNA and Y RNA)</span
          >
          more than 2,030 high-quality small RNA-seq datasets of EV samples from 24 human tissues of different conditions (disease and
          normal). In addition, EVAtlas provides modules for user to explore the conditional expression profiles of ncRNAs, miRNA targets,
          miRNA related drugs, miRNA regulated pathways, miRNA functions, and related publications.
        </p>
      </mat-card-content>
    </mat-card>

    <!-- ncRNA types -->
    <mat-card fxFlex appMaterialElevation defaultElevation="4" raisedElevation="4">
      <mat-card-header>
        <mat-card-title>ncRNAs</mat-card-title>
      </mat-card-header>
      <mat-card-content>
        <ol>
          <li>
            <p>
              <strong>miRNA: </strong>A microRNA (abbreviated miRNA) is a small non-coding RNA molecule (containing about 22 nucleotides)
              found in plants, animals and some viruses, that functions in RNA silencing and post-transcriptional regulation of gene
              expression. miRNAs function via base-pairing with complementary sequences within mRNA molecules. As a result, these mRNA
              molecules are silenced, by one or more of the following processes: (1) Cleavage of the mRNA strand into two pieces, (2)
              Destabilization of the mRNA through shortening of its poly(A) tail, and (3) Less efficient translation of the mRNA into
              proteins by ribosomes.
            </p>
          </li>
          <li>
            <p>
              <strong>piRNA: </strong>Piwi-interacting RNA (piRNA) is the largest class of small non-coding RNA molecules expressed in
              animal cells. piRNAs form RNA-protein complexes through interactions with piwi-subfamily Argonaute proteins. These piRNA
              complexes are mostly involved in the epigenetic and post-transcriptional silencing of transposable elements and other spurious
              or repeat-derived transcripts, but can also be involved in the regulation of other genetic elements in germ line cells.
            </p>
          </li>
          <li>
            <p>
              <strong>rRNA: </strong>Ribosomal ribonucleic acid (rRNA) is a type of non-coding RNA which is the primary component of
              ribosomes, essential to all cells. rRNA is a ribozyme which carries out protein synthesis in ribosomes. Ribosomal RNA is
              transcribed from ribosomal DNA (rDNA) and then bound to ribosomal proteins to form small and large ribosome subunits. rRNA is
              the physical and mechanical actor of the ribosome that forces transfer RNA (tRNA) and messenger RNA (mRNA) to process and
              translate the latter into proteins.[1] Ribosomal RNA is the predominant form of RNA found in most cells; it makes up about 80%
              of cellular RNA despite never being translated into proteins itself. Ribosomes are composed of approximately 60% rRNA and 40%
              ribosomal proteins by mass.
            </p>
          </li>
          <li>
            <p>
              <strong>tRNA: </strong>A transfer RNA (abbreviated tRNA and formerly referred to as sRNA, for soluble RNA) is an adaptor
              molecule composed of RNA, typically 76 to 90 nucleotides in length, that serves as the physical link between the mRNA and the
              amino acid sequence of proteins. tRNA does this by carrying an amino acid to the protein synthetic machinery of a cell
              (ribosome) as directed by the complementary recognition of a 3-nucleotide sequence (codon) in a messenger RNA (mRNA) by a
              3-nucleotide sequence (anticodon) of the tRNA. As such, tRNAs are a necessary component of translation, the biological
              synthesis of new proteins in accordance with the genetic code.
            </p>
          </li>
          <li>
            <p>
              <strong>Y RNA: </strong>Y RNAs are small non-coding RNAs. They are components of the Ro60 ribonucleoprotein particle which is
              a target of autoimmune antibodies in patients with systemic lupus erythematosus. They are also reported to be necessary for
              DNA replication through interactions with chromatin and initiation proteins. However, mouse embryonic stem cells lacking Y
              RNAs are viable and have normal cell cycles. Humans appear to have four Y RNAs, named hY1, hY3, hY4 and hY5 and also a large
              number of pseudogenes.
            </p>
          </li>
          <li>
            <p>
              <strong>snRNA: </strong>Small nuclear RNA (snRNA) is a class of small RNA molecules that are found within the splicing
              speckles and Cajal bodies of the cell nucleus in eukaryotic cells. The length of an average snRNA is approximately 150
              nucleotides. They are transcribed by either RNA polymerase II or RNA polymerase III. Their primary function is in the
              processing of pre-messenger RNA (hnRNA) in the nucleus. They have also been shown to aid in the regulation of transcription
              factors (7SK RNA) or RNA polymerase II (B2 RNA), and maintaining the telomeres.
            </p>
          </li>
          <li>
            <p>
              <strong>snoRNA: </strong>Small nucleolar RNAs (snoRNAs) are a class of small RNA molecules that primarily guide chemical
              modifications of other RNAs, mainly ribosomal RNAs, transfer RNAs and small nuclear RNAs. There are two main classes of
              snoRNA, the C/D box snoRNAs, which are associated with methylation, and the H/ACA box snoRNAs, which are associated with
              pseudouridylation. SnoRNAs are commonly referred to as guide RNAs but should not be confused with the guide RNAs that direct
              RNA editing in trypanosomes.
            </p>
          </li>
        </ol>
      </mat-card-content>
    </mat-card>

    <!-- Functional modules -->
    <mat-card fxFlex appMaterialElevation defaultElevation="4" raisedElevation="4">
      <mat-card-header>
        <mat-card-title>Function modules</mat-card-title>
      </mat-card-header>
      <mat-card-content>
        <ol>
          <li>
            <p>
              <a routerLink="/samples"><strong>Sample module:</strong></a> helps users to search the project and sample information of sEV
              or lEV in conditions , and browse their component ratio, heatmap and expression profiles of 7 types of ncRNAs.
            </p>
          </li>
          <li>
            <p>
              <a routerLink="/rna"><strong>Expression module:</strong></a> display the expression distribution of ncRNAs in collected
              samples and tissues, and provides the basic information and expressin profiles for ncRNAs across samples and conditions. For
              miRNAs, users can browsed comprehensive miRNA infomation, contain miRNA basic infomation, miRNA expression profiles, miRNA
              related functions, miRNA related TCGA expression, miRNA related drug and miRNA related target genes.
            </p>
          </li>

          <li>
            <p>
              <a routerLink="/specific"><strong>Specific Expression module:</strong></a> users can browse the specific expression patterns
              of ncRNAs(miRNA, piRNA and others) in conditions.
            </p>
          </li>
          <!-- <li>
            <p>
              <a routerLink="/samples"><strong>xxx1 module:</strong></a> helps users to search the project and sample information of
              Exosomes or Microvesicles in conditions , and browse their component ratio, heatmap and expression profiles of 8 types of
              ncRNAs.
            </p>
          </li>
          <li>
            <p>
              <a routerLink="/samples"><strong>xxx2 module:</strong></a> helps users to search the project and sample information of
              Exosomes or Microvesicles in conditions , and browse their component ratio, heatmap and expression profiles of 8 types of
              ncRNAs.
            </p>
          </li> -->
        </ol>
      </mat-card-content>
    </mat-card>

    <!-- Data collection -->
    <mat-card fxFlex appMaterialElevation defaultElevation="4" raisedElevation="4">
      <mat-card-header>
        <mat-card-title>Data collection and process</mat-card-title>
      </mat-card-header>
      <mat-card-content>
        <h3>Data collection</h3>
        <p>
          We used keywords "(((extracellular vesicle OR exosome OR microvesicle OR EV) AND (smRNA-seq OR miRNA-seq OR small RNA-seq))) AND
          (Homo sapiens)" to retrieved human EV related smRNA-seq datasets from NCBI
          <a href="https://www.ncbi.nlm.nih.gov/gds/" target="_blank">GEO</a> and
          <a href="https://www.ncbi.nlm.nih.gov/sra/?term=" target="_blank">SRA</a> databases. Then, we manually reviewed the study design
          or the related paper to label each dataset with sEV or lEV based on the exact EV size or type description. Finally, we obtained
          2030 high quality EV datasets (1506 sEV and 524 lEV datasets) from 57 studies. For better meta-analysis, all the datasets were
          well-curated and classified into 24 conditions, 8 sources and over 40 diseases. The criteria used to distinguish the belonging of
          given samples (sEVs or lEVs) were as following:
        </p>
        <ol>
          <li>samples with clearly defined of the belonging were obeyed to the original one;</li>
          <li>
            samples with detailed description of isolation method were adapted to the criteria of International Society for Extracellular
            Vesicles (ISEV);
          </li>
          <li>samples extracted by commercial kits were classified according to the introduction of manufacture;</li>
          <li>
            samples out of above categories were defined as sEVs or lEVs based on the word frequency count of "exosomes" or "microvesicles";
            Samples that cannot be grouped were discarded.
          </li>
        </ol>
        <h3>Pre-process</h3>
        <p>
          Furthermore, we examined the quality of sequencing data of each dataset using similar methods as
          <a href="http://bioinfo.life.hust.edu.cn/EVmiRNA#!/" target="_blank">EVmiRNA</a> described in our previous study. In brief,
          <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">FastQC</a>
          (v0.11.5) was used for data quality control (QC), and
          <a href="http://www.usadellab.org/cms/?page=trimmomatic" target="_blank">trimmomatic</a> (v0.36) were employed for adapter
          sequence removal and FASTQ trimming to obtain high-quality clean reads.
          <a href="http://bowtie-bio.sourceforge.net/index.shtml" target="_blank">Bowtie</a> (v1.2.1) was used to align clean reads to human
          reference genome GRCh38. Samples with clean reads less than 5 million or the predominant reads out of the range of 15-50 nt or an
          alignment ratio less than 40% were discarded. Finally, 2030 EV datasets (1506 sEVs and 524 lEVs datasets) from 24 conditions with
          40 diseases were kept for further analyses.
        </p>

        <h3>Read mapping and expression estimation of ncRNA</h3>
        <p>
          Before reads mapping, we obtained the reference sequences for the 7 types of ncRNAs from different authoritative databases. Then
          we used the Bowtie software and the following <strong>R</strong>eads <strong>D</strong>ynamic <strong>A</strong>ssignment
          <strong>A</strong>lgorithm (RDAA) for reads mapping. We summarized RDAA in following steps:
        </p>
        <ol>
          <li>
            Each sample clean reads are mapped (one mismatch allowed) to seven ncRNA references simultaneously, and gather all ncRNAs mapped
            reads together to prepare assignment;
          </li>
          <li>
            Reads with 0 or 1 mismatch only mapped to one ncRNA type are used to calculate ncRNA abundance;
          </li>
          <li>
            the reads with 0 mismatch mapped to one ncRNA type are regarded as confident reads which are used to calculate ncRNA proportion
            in EV, which would be regarded as reference ncRNA abundance for later assignment;
          </li>
          <li>
            multi-ncRNA mapped reads with 0 or 1 mismatch are assigned to exact one ncRNA type according to reference ncRNA abundance
            priority;
          </li>
          <li>
            Final ncRNA abundance is determined by the assigned ncRNA mapped reads.
          </li>
        </ol>
        <p>The RDAA mapping strategy is shown in the workflow as follow:</p>
        <img mat-card-image class="zm" [src]="assets + '/imgs/' + 'tag-assignment.svg'" />
        <p>
          The normalization method of expressed reads per million (RPM) was employed to estimate the expression level of ncRNAs using the
          total mapped reads. The reference sequences of miRNAs were downloaded from
          <a href="http://www.mirbase.org/" target="_blank">miRBase V22</a>, while the references of tRNA and snoRNA are obtained from
          <a href="http://scottgroup.med.usherbrooke.ca/snoDB/" target="_blank">snoDB</a> and
          <a href="http://gtrnadb.ucsc.edu/" target="_blank">GtRNAdb</a>, respectively. Other reference sequences, such as piRNA, snRNA,
          rRNA, and YRNA were parsed from human genome annotation of NCBI
          <a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank">GenBank</a> or
          <a href="https://rnacentral.org/" target="_blank">RNACentral</a>.
        </p>
        <h3>Specifically expressed ncRNAs</h3>
        <p>
          To detect specifically expressed ncRNAs, the expression level of a ncRNA in the given condition was represented by its median. The
          specifically highly expressed ncRNAs from sEVs (22 conditions) and lEVs (7 conditions) were analyzed separately with
          <a href="http://bioinfo.life.hust.edu.cn/SEGtool/" target="_blank">SEGtool</a>.
        </p>
        <h3>Functional annotation for ncRNAs and their expression profiles in cancers</h3>
        <p>
          The functional annotation of ncRNAs were integrated into EVAtlas, in which, different types of ncRNAs have their distinct
          annotations. The basic information, such as gene location and potential functions, was extracted from the corresponding reference
          database of each ncRNA type and incubated into EVAtlas. For example, the basic information and functions of rRNAs were collected
          from RNA central. For miRNAs, EVAtlas collected the their <span style="color: red;">related functions</span> from
          <a href="https://pubmed.ncbi.nlm.nih.gov/" target="_blank">PubMed</a>,
          <span style="color: red;">miRNA-targets relations</span> from
          <a href="http://bioinfo.life.hust.edu.cn/FFLtool#!/" target="_blank">FFLtool</a>,
          <span style="color: red;"> drugs correlation</span> from SM2miR <a href="http://www.jianglab.cn/SM2miR/" target="_blank"></a>,
          their <span style="color: red;"> expression profiles</span> in cancers from
          <a href="http://bioinfo.life.hust.edu.cn/web/GSCALite/" target="_blank">GSCALite</a>. For snoRNA and tRNA, their expression
          profiles were collected from <a href="">SNORic</a> and <a href="">TRic</a>, while their potential target genes were incubated from
          snoDB and GtRNAdb.
        </p>
      </mat-card-content>
    </mat-card>

    <!-- sEV and lEV mapping rate -->
    <div fxFlex fxLayout.lt-lg="column" fxLayoutGap="10px">
      <mat-card fxFlex appMaterialElevation defaultElevation="4" raisedElevation="4">
        <mat-card-header>
          <mat-card-title>{{ exoMappingRateTitle }}</mat-card-title>
        </mat-card-header>
        <mat-card-content>
          <div *ngIf="!exoMappingRate">
            <mat-progress-bar mode="query" color="warn"></mat-progress-bar>
          </div>
          <div echarts [options]="exoMappingRate" class="chart-hight"></div>
        </mat-card-content>
      </mat-card>
      <br />

      <mat-card fxFlex appMaterialElevation defaultElevation="4" raisedElevation="4">
        <mat-card-header>
          <mat-card-title>{{ mvMappingRateTitle }}</mat-card-title>
        </mat-card-header>
        <mat-card-content>
          <div class="spinner-container" *ngIf="!mvMappingRate">
            <mat-progress-bar mode="query" color="warn"></mat-progress-bar>
          </div>
          <div echarts [options]="mvMappingRate" class="chart-hight"></div>
        </mat-card-content>
      </mat-card>
    </div>

    <!-- EV isolation techniques -->
    <mat-card fxFlex appMaterialElevation defaultElevation="4" raisedElevation="4">
      <mat-card-header>
        <mat-card-title>EV isolation techniques</mat-card-title>
      </mat-card-header>
      <mat-card-content>
        <ol>
          <li>
            <!-- <p> -->
            <!-- We downloaded a total of 2,831 small RNA sequencing datasets of exosomes and microvesicles samples
              <a href="https://www.ncbi.nlm.nih.gov/sra/" target="_blank">NCBI SRA database</a> (up to 30/05/2020). Then, we manually
              examined the sources of samples and their isolation method to ensure their categories (exosome or microvesicle or others)
              according to Mateescu's paper (details in the following picture). Finally, after quality control, we obtained
              <strong style="color: DarkViolet;">2,271</strong> samples from 24 human conditions and 40 diseases. Furthermore,.
            </p> -->
            <!-- <h3>EV isolation techniques</h3> -->
            <!-- <img mat-card-image class="evi_func" [src]="assets + '/imgs/' + 'evisolated.jpg'" /> -->
            <!-- </p> -->

            <p>
              <cite>
                Mateescu, Bogdan, et al. "Obstacles and opportunities in the functional analysis of extracellular vesicle RNA-an ISEV
                position paper." Journal of extracellular vesicles 6.1 (2017): 1286095.</cite
              >
            </p>
            <ul>
              <li>
                (a) Differential centrifugation is the sequential pelleting of particles with decreasing sedimentation coefficients.
                Typically 2000 g is used to pellet large EVs, 10,000-20,000 g to pellet middle-sized EVs (green) (MVs), and finally ~100,000
                g to pellet the smallest EVs (different EV subpopulations are indicated in grey and orange) (Exosomes). At these high
                g-forces, complexes of soluble proteins (black dots) may also sediment.
              </li>
              <li>
                (b) Lipids have a density that is approximately 1 g cm-3, while proteins and RNA have a higher density (>1.3 g cm-3).
                Therefore density gradients can be used to separate subpopulations of EVs with different ratio of lipids, RNA, and proteins.
                Moreover, these gradients can be used to purify vesicles away from soluble proteins, RNA, and protein-RNA complexes as the
                latter structures will not float at the same density as the lipid containing EVs.
              </li>
              <li>
                (c) Size exclusion chromatography separates particles based on their size, by trapping the smaller molecules (such as
                proteins and protein complexes) in the pores. The larger molecules (such as EVs) are too large to enter the pores and will
                elute first.
              </li>
              <li>
                (d) Precipitation of EV from cell culture medium or body fluids is based on volume-excluding polymers such as polyethylene
                glycol (PEG) with which biological materials such as proteins and EVs are precipitated from the solution.
              </li>
              <li>
                (e) (Immuno-)affinity capture isolates vesicles using beads coated with antibodies or proteins (such as heparin) with
                affinity for an EV transmembrane protein. Vesicles displaying the protein of interest will bind to the beads and can thereby
                be isolated from the vesicle-containing solution.
              </li>
            </ul>
            <img mat-card-image class="evi_func" [src]="assets + '/imgs/' + 'evisolated.jpg'" />
            <!-- <p><b>Small RNA sequencing data mapping Strategy</b></p>
            <p>
              Datasets with ambiguous description of experiment design and less raw reads were discarded. Subsequently, for each dataset,
              reads with the multiple N bases (>1) or unexpected length after adapter remove (> 45 nt or < 15 nt) or with low quality (mean
              quality <=20) were filtered, and dataset with less reads (< 1M) after the procedure were discarded. Finally, 2271 EV datasets
              were kept for further analysis, which including 1,660 exosome and 563 MV samples from 23 human tissues(organs) and 40 diseases
              (Table 1). Clean reads were mapped to human pre-miRNA sequences from miRBase V22 and other ncRNAs from Genebank using bowtie
              (1 mismatch allowed) to identify known ncRNAs and estimate their abundance. The normalization method of RPM (expressed Reads
              Per Million reads) were employed to assess the expression level of ncRNAs across samples and conditions.
            </p> -->
          </li>
          <li>
            <p>
              <cite>
                Thery, Clotilde, et al. "Minimal information for studies of extracellular vesicles 2018 (MISEV2018): a position statement of
                the International Society for Extracellular Vesicles and update of the MISEV2014 guidelines." Journal of extracellular
                vesicles 7.1 (2018): 1535750.</cite
              >
            </p>
          </li>
          <!-- <img mat-card-image class="zm" [src]="assets + '/imgs/' + 'zm.png'" /> -->
          <!-- <li>
            <p>
              Annotation data source: The annotation file of piRNAs was downloaded from piRNAQuest database, which the snoRNA annotation was
              obtained from snoRNABase. The annotation information of snRNA, YRNA and pRNA were parsed from ENSEMBL v86 gtf file. The rRNA
              and tRNA information were incubated from NCBI GeneBank database. The miRNA basic information was collected from miRbase.
              Pathway information was collected from KEGG database, and use Hypergeometric p-value to filter miRNA pathway pairs.
              Publication information was collected from NCBI pubmed database. miRNA function was collected from NCBI RIF database. miRNA
              Drug information was collected from SM2miR (Small molecular miRNA database). miRNA targets was from TargetScan, miRanda,
              TarBase, miRTarBase, miR2Disease, miRecords and miRWalk.
            </p>
            <b>RNA types & annotation</b>
            <img mat-card-image class="df" [src]="assets + '/imgs/' + 'data_function_static.png'" />
          </li> -->
          <li>
            <p>
              We applied SEGtool(exp_cutoff = 3,multi_cpu = 4, detect_mod=3) to identify Specific miRNAs of cancer and tissue. We used miRNA
              expression cutoff RPM >= 100 to identify specific miRNAs in cancer group and tissue group. We found that in cancer group, sEV
              can identify more specific miRNAs.
            </p>
            <p>
              <b>Specific miRNA</b>
            </p>
            <table mat-table [dataSource]="dataSource" class="mat-elevation-z8">
              <ng-container matColumnDef="Source">
                <th mat-header-cell *matHeaderCellDef>No.</th>
                <td mat-cell *matCellDef="let element">{{ element.Source }}</td>
              </ng-container>

              <ng-container matColumnDef="EV_type">
                <th mat-header-cell *matHeaderCellDef>EV type</th>
                <td mat-cell *matCellDef="let element">{{ element.EV_type }}</td>
              </ng-container>

              <ng-container matColumnDef="Cancer_Source_type">
                <th mat-header-cell *matHeaderCellDef>Source type</th>
                <td mat-cell *matCellDef="let element">{{ element.Cancer_Source_type }}</td>
              </ng-container>

              <ng-container matColumnDef="Specific_miRNA_counts">
                <th mat-header-cell *matHeaderCellDef>Specific miRNA counts</th>
                <td mat-cell *matCellDef="let element">{{ element.Specific_miRNA_counts }}</td>
              </ng-container>

              <ng-container matColumnDef="Specific_piRNA_counts">
                <th mat-header-cell *matHeaderCellDef>Specific piRNA counts</th>
                <td mat-cell *matCellDef="let element">{{ element.Specific_piRNA_counts }}</td>
              </ng-container>

              <ng-container matColumnDef="Specific_others_counts">
                <th mat-header-cell *matHeaderCellDef>Specific others counts</th>
                <td mat-cell *matCellDef="let element">{{ element.Specific_others_counts }}</td>
              </ng-container>

              <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
              <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
            </table>
          </li>
        </ol>
      </mat-card-content>
    </mat-card>
  </div>

  <!-- Acknowledgements -->
  <mat-card appMaterialElevation defaultElevation="4" raisedElevation="4">
    <mat-card-header>
      <mat-card-title>Acknowledgements and external links</mat-card-title>
    </mat-card-header>
    <mat-card-content>
      <p>Many thanks to the data producers.</p>
      <!-- <ul>
        <li> -->
      <a href="http://www.journalofextracellularvesicles.net/" target="_blank"
        ><img class="thx" mat-card-image [src]="assets + '/imgs/' + 'JEV.png'"
      /></a>
      <!-- </li>
        <li> -->
      <a href="http://www.isev.org/" target="_blank"><img class="thx" mat-card-image [src]="assets + '/imgs/' + 'ISEV.png'" /></a>
      <!-- </li>
        <li> -->
      <a href="http://www.exrna.org/" target="_blank"><img class="thx" mat-card-image [src]="assets + '/imgs/' + 'Exrna.png'" /></a>
      <!-- </li>
      </ul> -->
      <h3>EVAtlas data contributors</h3>
      <!-- <p>test: {{ sampleSource }}</p> -->
      <table mat-table [dataSource]="sampleSource" class="mat-elevation-z8">
        <ng-container matColumnDef="gse_id">
          <th mat-header-cell *matHeaderCellDef>GSE ID</th>
          <td mat-cell *matCellDef="let element">
            <a href="{{ element.link }}" target="_blank">{{ element.gse_id }}</a>
          </td>
        </ng-container>

        <ng-container matColumnDef="Organization">
          <th mat-header-cell *matHeaderCellDef>Organization</th>
          <td mat-cell *matCellDef="let element">{{ element.Organization }}</td>
        </ng-container>

        <ng-container matColumnDef="Contributor">
          <th mat-header-cell *matHeaderCellDef>Contributor</th>
          <td mat-cell *matCellDef="let element">{{ element.Contributor }}</td>
        </ng-container>

        <!-- <ng-container matColumnDef="Overall_design">
          <th mat-header-cell *matHeaderCellDef>Overall design</th>
          <td mat-cell *matCellDef="let element">{{ element.Overall_design }}</td>
        </ng-container>

        <ng-container matColumnDef="Summary">
          <th mat-header-cell *matHeaderCellDef>Summary</th>
          <td mat-cell *matCellDef="let element">{{ element.Summary }}</td>
        </ng-container> -->

        <ng-container matColumnDef="pubmed_id">
          <th mat-header-cell *matHeaderCellDef>PMID</th>
          <td mat-cell *matCellDef="let element">
            <a href="{{ element.link_ref }}" target="_blank">{{ element.pubmed_id }}</a>
          </td>
        </ng-container>

        <!-- <ng-container matColumnDef="link">
          <th mat-header-cell *matHeaderCellDef>Detail</th>
          <td mat-cell *matCellDef="let element">
            <a href="{{ element.link }}" target="_blank"><mat-icon style="color: coral;">list</mat-icon></a>
          </td>
        </ng-container> -->

        <tr mat-header-row *matHeaderRowDef="displayedSamples"></tr>
        <tr mat-row *matRowDef="let row; columns: displayedSamples"></tr>
      </table>
    </mat-card-content>
  </mat-card>

  <div fxFlex="none">
    <app-footer></app-footer>
  </div>
</div>
